test/tc_selinux_profilebindings_test.go

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_test
+
+import (
+	"fmt"
+	"strings"
+)
+
+func (e *e2e) testCaseSelinuxProfileBinding([]string) {
+	e.selinuxOnlyTestCase()
+	const selinuxTestProfileName = "testpolicy"
+	const selinuxBindingName = "profile-binding-selinux"
+	const testPodName = "selinux-profile-test-pod"
+	selinuxTestProfile := fmt.Sprintf(`
+apiVersion: security-profiles-operator.x-k8s.io/v1alpha2
+kind: SelinuxProfile
+metadata:
+  name: %s
+spec:
+  allow:
+    var_log_t:
+      dir:
+        - open
+        - read
+        - getattr
+        - lock
+        - search
+        - ioctl
+        - add_name
+        - remove_name
+        - write
+      file:
+        - getattr
+        - read
+        - write
+        - append
+        - ioctl
+        - lock
+        - map
+        - open
+        - create
+      sock_file:
+        - getattr
+        - read
+        - write
+        - append
+        - open
+`, selinuxTestProfileName)
+
+	selinuxBinding := fmt.Sprintf(`
+apiVersion: security-profiles-operator.x-k8s.io/v1alpha1
+kind: ProfileBinding
+metadata:
+  name: %s
+spec:
+  profileRef:
+    kind: SelinuxProfile
+    name: %s
+  image: quay.io/wenshen/spo_test:selinux_busybox
+`, selinuxBindingName, selinuxTestProfileName)
+
+	testPod := fmt.Sprintf(`
+apiVersion: v1
+kind: Pod
+metadata:
+  name: %s
+spec:
+  initContainers:
+  - image: quay.io/wenshen/spo_test:selinux_busybox
```

Comment:
This is OK in a follow-up patch (let's merge this one..) but we generally prefer to keep our test images in our quay.io repo. Is the Dockefile for the image available? Or is that maybe just some generic busybox image?

Reply:
I have swapped it to busybox image docker image